build.go

```diff
@@ -241,6 +233,18 @@ func main() {
 	}
 }
 
+// set GOPATH correctly with the assumption that we are
+// in $GOPATH/src/github.com/syncthing/syncthing.
+func setGoPath() {
```

Comment:
It needs to start with `setGoPath` as the first word otherwise vet should give a warning.